File: pubnub-gson/pubnub-gson-api/src/main/java/com/pubnub/api/java/endpoints/channel_groups/DeleteChannelGroup.java
package com.pubnub.api.java.endpoints.channel_groups;

import com.pubnub.api.java.endpoints.Endpoint;
import com.pubnub.api.models.consumer.channel_group.PNChannelGroupsDeleteGroupResult;

public interface DeleteChannelGroup extends Endpoint<PNChannelGroupsDeleteGroupResult> {
    DeleteChannelGroup channelGroup(String channelGroup);
}

File: pubnub-kotlin/pubnub-kotlin-api/src/commonMain/kotlin/com/pubnub/api/endpoints/channel_groups/DeleteChannelGroup.kt
package com.pubnub.api.endpoints.channel_groups

import com.pubnub.api.models.consumer.channel_group.PNChannelGroupsDeleteGroupResult
import com.pubnub.kmp.PNFuture

/**
 * @see [PubNub.deleteChannelGroup]
 */
expect interface DeleteChannelGroup : PNFuture<PNChannelGroupsDeleteGroupResult>
